ui/src/redux/modules/visualise.js

```diff
@@ -449,10 +451,20 @@ export const getDateRange = (period) => {
   }
 };
 
-const MAX_NAME_LENGTH = 12;
+const MAX_NAME_LENGTH = 17;
 export const trimName = (name, length = MAX_NAME_LENGTH) => {
-  if (name.length <= length) return name;
-  return `...${name.substr(-length)}`;
+  let formattedName;
+  if (name.length >= length) {
+    switch (true) {
```

Comment:
Use if blocks here instead of a switch true please, the switch is totally redundant and breaking out of the cases is making this code unnecessarily verbose.

Reply:
Not sure if this isn't pushed, but this hasn't changed yet